addons-example.yml

```diff
@@ -90,8 +92,11 @@ my-rds-db:
   environments:
     default:
       plan: small-ha
-      deletion-policy: Retain # Delete|Retain|Snapshot
```

Comment:
Do we still need this policy? Or does protection take priority?

Reply:
I can see both are set for the `production` env.

Reply:
The `deletion-policy` is specific to preventing CloudFormation deleting a resource. Presumably `deletion-protection` would also stop that scenario but I'm not sure. There may be situations where you'd want one vs the other.

Reply:
It does feel a bit weird having different settings overlapping like this, but I think we can thank AWS for that.

Reply:
It is strange but I don't think there's much we can do about it. I tested it and `deletion-protection` does stop CloudFormation from deleting the database but it gets stuck in _update-cleanup-in-progress_ for ages.

Also `deletion-protection` is only an option for databases so `deletion-policy` would still have to be set on the other addons anyway.